### PR TITLE
Update: Update datelines to pass accessibility(fixes #268)

### LINKED ISSUE #268: Update last load/today and DST indicators to pass accessibility
Current Carbon Last Load (yellow) and DST (gray)
image

Update to orange _square_ and line and blue/gray _diamond_ and line:
image
image

NEW blue/gray: #405978
NEW orange: #C97318

Instead of using the triangle for DST, we've determined to use the diamond. You can use the existing regular diamond icon from Carbon with the new color.

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
Fixes: #268 

**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [ ] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [x] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [X] No

**What changes did you make? (Give an overview)**
- Made changes to dateline in all graphs to meet accessibility requirements.

**Dev site Example path**
- https://engineering.cerner.com/carbon-graphs/#/line/timeseries-dateline
- https://engineering.cerner.com/carbon-graphs/#/paired-result/timeseries-dateline
- https://engineering.cerner.com/carbon-graphs/#/bar/timeseries-dateline
- https://engineering.cerner.com/carbon-graphs/#/scatter/timeseries-dateline
- https://engineering.cerner.com/carbon-graphs/#/gantt/eventline

**Add any screenshots**
**Before**
image

image

image

image

image

**After**

image

image

image

image

image

### CODE REVIEW

**Comment by AshishMotanamGurunadham** [comment]
@jeremyfuksa Added dev example path in description.

**Comment by jeremyfuksa** [comment]
Were you given advice on whether the actual line must also pass contrast? Or is it just the dateline marker? Because while the marker passes, the opacity in the actual line only yields 1.41:1 contrast.

**Comment by AshishMotanamGurunadham** [comment]
@jeremyfuksa [issue](https://github.com/cerner/carbon-graphs/issues/268) indicates to change both line and marker in dateline.

**Comment by jeremyfuksa** [comment]
Right. And you did that, but the line itself is still not passing contrast because of its opacity. Let's set the opacity of the actual dateline to 100% so that it passes contrast tests.

**Review by jeremyfuksa — APPROVED** [review]

File: dev/examples/controls/bar.js [hunk 1]
```diff
@@ -275,7 +275,7 @@ export const renderBarTimeSeriesWithDateline = (id) => {
             label: {
                 display: "Action Date"
             },
-            color: "#FFDF00",
+            color: "#C97318",
```
Comment (Veera81) [inline dev/examples/controls/bar.js:278]: Is this color is present in the carbon color constants? If yes, can we use it from constants in the below files?

**Review by Veera81 — CHANGES_REQUESTED** [review]
Is this change to reflect only the dev site? How does it affect the consumer?

File: dev/examples/controls/bar.js (hunk 1 above)
Comment (AshishMotanamGurunadham) [reply dev/examples/controls/bar.js:278]: No it is not present in constants.

Comment (Veera81) [reply dev/examples/controls/bar.js:306]: Is it worth it to move the color constants to carbon color constants? Is this only used for dev site? or used by consumer also?

Comment (AshishMotanamGurunadham) [reply dev/examples/controls/bar.js:306]: @Veera81 It is not worth to move those to color constants because those are only used in dev site.

**Review by Veera81 — APPROVED** [review]

**Comment by Veera81** [comment]
@AshishMotanamGurunadham Is this change to reflect only the dev site? How does it affect the consumer?

**Comment by AshishMotanamGurunadham** [comment]
> @AshishMotanamGurunadham Is this change to reflect only the dev site? How does it affect the consumer?

These changes are only for dev site.

**Review by sdadn — CHANGES_REQUESTED** [review]

File: dev/examples/controls/gantt.js [hunk 2]
```diff
@@ -539,7 +539,7 @@ export const renderGanttPanning = (id) => {
             label: {
                 display: "Release A"
             },
-            color: Carbon.helpers.COLORS.GREEN,
+            color: "#C97318",
```
[the remark this answers is not in the document]
Comment (AshishMotanamGurunadham) [reply dev/examples/controls/gantt.js:542]: Yes Saad.

**Review by sdadn — APPROVED** [review]

**Pull request merged**